lists/listed.json

```
@@ -76,6 +76,7 @@
 		"0x956F47F50A910163D8BF957Cf5846D573E7f87CA",
 		"0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B",
 		"0xCFEAead4947f0705A14ec42aC3D44129E1Ef3eD5",
-		"0xB62132e35a6c13ee1EE0f84dC5d40bad8d815206"
+		"0xB62132e35a6c13ee1EE0f84dC5d40bad8d815206",
+		"0xbae28251b2a4e621aa7e20538c06dee010bc06de"
```

```suggestion
		"0xbAe28251B2a4E621aA7e20538c06DEe010Bc06DE"
```
Has to be checksummed here. Also, I don't see anything at this address. Is the token deployed on mainnet?

---

It's a polygon token. Is this list only mainnet?

---

I believe there's only one list for "production" networks, so that should be fine. Does the token need to be in index.json too, though? Add the Coingecko feed too (is it https://www.coingecko.com/en/coins/defidollar)?

---

@EndymionJkb on Coingecko it's [dHEDGE Stablecoin Yield](https://www.coingecko.com/en/coins/dhedge-stablecoin-yield)